src/main/client_context.cpp

```diff
@@ -60,6 +61,9 @@ struct ActiveQueryContext {
 ClientContext::ClientContext(shared_ptr<DatabaseInstance> database)
     : db(move(database)), transaction(db->GetTransactionManager(), *this), interrupted(false),
       client_data(make_unique<ClientData>(*this)) {
+	for (auto &functions : EXTENSION_FUNCTIONS) {
```

Comment:
This seems quite wasteful, especially as the list of functions will grow and lookups are rarely needed. Can we instead do a lookup in the list of extension names when this is actually required instead, similar to how the default functions and default views work? Perhaps the names can be stored in alphabetical order so a binary search can be done?